tests/loki/iofq_test.py

```diff
@@ -280,13 +280,13 @@ def test_phi_with_gravity():
     pipeline = make_workflow()
     pipeline[BeamCenter] = _compute_beam_center()
     pipeline[CorrectForGravity] = False
-    data_no_grav = pipeline.compute(
-        CleanWavelengthMasked[SampleRun, Numerator]
-    ).flatten(to='pixel')
+    data_no_grav = pipeline.compute(CleanWavelength[SampleRun, Numerator]).flatten(
+        to='pixel'
+    )
     graph_no_grav = pipeline.compute(ElasticCoordTransformGraph)
     pipeline[CorrectForGravity] = True
     data_with_grav = (
-        pipeline.compute(CleanWavelengthMasked[SampleRun, Numerator])
+        pipeline.compute(CleanWavelength[SampleRun, Numerator])
         .flatten(to='pixel')
         .hist(wavelength=sc.linspace('wavelength', 1.0, 12.0, 101, unit='angstrom'))
     )
```

Comment:
Is it worth adding a test which sort of simulates the streaming? Basically make two chunks of data that are accumulated? I'm not sure what you would test though? Just that it doesn't fail?

Or does this belong more in the data streaming (i.e. in a different repository)?

Reply:
In any case, it does not belong into this PR. And since even before restructure streaming was possible (it just recomputed more than strictly necessary) it would require mocking some workflow components and adding call counters for the tests. Such tests exist in the `ess.reduce.streaming` module.